project_scrum/views/project_project.xml

```diff
@@ -58,6 +58,22 @@
                     <field name="meeting_count" widget="statinfo" />
                 </button>
             </button>
+            <xpath
+                expr="//field[@name='task_count']/parent::button"
+                position="replace">
+                <button
+                    class="oe_stat_button"
+                    type="action"
+                    attrs="{'invisible':[('use_tasks','=', False)]}"
+                    name="%(action_show_scrum_tasks)d"
```

Comment:
his doesn't do what's asked. you should display the tasks *of this project* with the scrum tree/form view. by just using the other action (which doesn't set a search_default_...), you will  show all tasks. i think it's possible to fix this by just setting the context attribute on the button, but not sure.
the error -proof way is to make a new action that has  a context key , which then will take context key and filter on project.  you should use active_id to pass to the context the current project.

that will work and obtain what you want. before doing that see if adding a context to the button works. so the context needs to have the current active ID, then you need to use that active id (of the current project ) to filter out results.

see if works on button and if not just make another action as described above.